osmosis-1/osmosis-1.assetlist.json

```diff
@@ -13566,6 +13566,31 @@
         "osmosis_unlisted",
         "osmosis-unlisted"
       ]
+    },
+    {
```

Comment:
Please do not edit this osmosis-1.assetlist.json file directly.

Reply:
Done sorry